build/android/patches/irrlicht-back_button.patch

```diff
@@ -1,5 +1,14 @@
---- irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp.orig	2016-05-02 04:32:21.281252634 +0300
-+++ irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp	2016-05-02 04:37:58.295589904 +0300
+--- irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp.orig	2015-08-29 15:43:09.000000000 +0300
++++ irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp	2016-05-13 21:36:22.880388505 +0300
+@@ -486,7 +486,7 @@
+ 				event.KeyInput.Char = 0;
+ 			}
+ 
+-			device->postEventFromUser(event);
++			status = device->postEventFromUser(event) ? 1 : 0;

```

Comment:
What does is this for?

Looking at the source code it seems to be to let the calling function know whether the event was absorbed or not. I *think* it's correct (your change) but I wonder why it wasn't always like this